classes/MockDisablerPHPUnit10.php

```diff
@@ -39,10 +46,16 @@ public function __construct(Deactivatable $deactivatable)
     public function notify(Finished $event) : void
     {
         $this->deactivatable->disable();
+        if ($this->callback !== null) {
+            call_user_func($this->callback, $this);
```

Comment:
```suggestion
            ($this->callback)($this);
```

just CS improvement...

I will test the actual fix later.